home.asm

```diff
@@ -56,9 +56,11 @@ INCLUDE "home/predef.asm"
 INCLUDE "home/window.asm"
 INCLUDE "home/flag.asm"
 
-Unreferenced_Function2ebb::
-	ld a, [wMonStatusFlags]
-	bit 1, a
+Unreferenced_CheckBPressedDebug::
+; Used in debug ROMs to walk through walls and avoid encounters.
```

Comment:
There seems to be no walk through walls code left, so you could mention pokegold-spaceworld here, like data/pokemon/unused_pic_banks.asm.

Reply:
:+1:

Reply:
This isn't specific to pokegold-spaceworld, as it's rather generic, and should exist in all of the Debug ROMs if I'm not mistaken.
I'm not sure what I'd mention about spaceworld here, as there's not much more to link to about it.